.github/workflows/r2-data-truth-hardening.yml

```
@@ -34,8 +34,28 @@ jobs:
+          --health-cmd "pg_isready -U skeldir_r2_test -d skeldir_r2_test"
+          --health-interval 5s
+          --health-timeout 5s
+          --health-retries 40
+          -c log_statement=all
+          -c log_destination=stderr
+          -c logging_collector=on
     env:
       ARTIFACTS_DIR: ${{ github.workspace }}/artifacts/runtime_r2/${{ github.sha }}
+      PGHOST: 127.0.0.1
+      PGPORT: "5432"
+      PGPASSWORD: ${{ env.POSTGRES_PASSWORD }}
```

actionlint (1.7.9)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)